src/amoc_throttle/amoc_throttle_controller.erl

```diff
@@ -181,7 +181,6 @@ handle_call({change_rate_gradually, Name, LowRate, HighRate,
 handle_call({stop, Name}, _From, State) ->
     case all_processes(Name, stop) of
         ok ->
-            pg2:delete(Name),
```

Comment:
is group removed in this case? can we recreate the group with the same name after this?

Reply:
https://github.com/erlang/otp/blob/4d2a17302a79a86b6c74c769b9fafd9fbd9be516/lib/kernel/src/pg.erl#L21-L53 Here's an explanation, non-existent and empty groups are treated the same in this new implementation, the reason being agreement in the case of coming back from a network split.